test/neubot/dash.cpp

```diff
@@ -0,0 +1,39 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
TODO: make sure that tests cover all possible errors and are up-to-date with respect to changes in the code (I am not even sure whether currently these tests compile cleanly).